lib/helpers.js

```diff
@@ -96,7 +96,8 @@ export function createExpectationMiddleware({
     chosen via some other means, e.g., via the request body, then the caller
     MUST provide the expected action and not rely on default behavior. */
     if(req.ezcap.expectedAction === undefined) {
-      req.ezcap.expectedAction = req.method === 'POST' ? 'write' : 'read';
+      req.ezcap.expectedAction =
+        req.method === 'POST' || req.method === 'DELETE' ? 'write' : 'read';
```

Comment:
I'm going to push a commit to change this up a bit.

Reply:
+1, and let's include `PUT` there as well.

Reply:
Yeah, I was already adding all common methods.